lib/_http_server.js

```diff
@@ -463,7 +463,7 @@ function connectionListenerInternal(server, socket) {
     server.maxHeaderSize || 0,
     server.insecureHTTPParser === undefined ?
       isLenient() : server.insecureHTTPParser,
-    server.headersTimeout || 0,
+    server.headersTimeout || 0
```

Comment:
Is there a way to avoid having to remove these trailing commas?

Reply:
Yes, we can specifically permit them with a rule setting. I didn't realize we were using trailing commas in function arguments. Will fix.

Reply:
OK, fixed up, please take a look.